packages/flutter_driver/lib/src/driver/web_driver.dart

```diff
@@ -55,12 +60,23 @@ class WebFlutterDriver extends FlutterDriver {
       'session-uri': Platform.environment['DRIVER_SESSION_URI'],
       'session-spec': Platform.environment['DRIVER_SESSION_SPEC'],
       'android-chrome-on-emulator': Platform.environment['ANDROID_CHROME_ON_EMULATOR'] == 'true',
+      'session-capabilities': Platform.environment['DRIVER_SESSION_CAPABILITIES'],
     };
     final FlutterWebConnection connection = await FlutterWebConnection.connect
       (hostUrl, settings, timeout: timeout);
     return WebFlutterDriver.connectedTo(connection);
   }
 
+  @override
+  Future<void> enableAccessibility() async {
+    if (!_accessibilityEnabled) {
+      await _connection._driver.execute(
+          'document.querySelector(\'flt-semantics-placeholder\').click();',
+          <String>[]);
+      _accessibilityEnabled = true;
+    }
```

Comment:
There are a few things we need to do here.
(1) This only works for desktop web (mobile web has another method which is very similar to this)
(2) If `flt-semantics-placeholder` which lives on the engine changes, this will break. It would be very hard to understand for the engine-developers. It is best to add a documentation and also a link to that code.

Btw, a question we can also click on this button by calling the following right?
```
webDriver.execute(
          'document.querySelector(\'flt-semantics-placeholder\').click();',
          <String>[]);
```

Reply:
Updated accordingly. Well create an issue for Mobile Web (in fact, it works on Android-Chrome).